config/environments/test.rb

```diff
@@ -49,4 +49,8 @@
 
   # Use separate cache stores for parallel_tests
   config.cache_store = :file_store, Rails.root.join("tmp", "cache", "paralleltests#{ENV['TEST_ENV_NUMBER']}")
+
+  # Disables automatic creation of the test DB so that it has to be explicitly
+  # generated before running tests.
+  config.active_record.maintain_test_schema = false
```

Comment:
> Right, I could 'enhance' the test task here. Or I could simply call Rake::Task['jenkins:db'] at the beginning of test_helper if the db hasn't been initialized yet. Otherwise the developer would need to run db:migrate before test on a new installation.

Enhancing the task would at least make it more consistent with how you've changed the jenkins task to work, which would be good.  I'm unsure if calling the rake task within the test helper would actually be soon enough to fix the issue, but I've not looked into it much.  If it works, it would again make the two more equivalent I suppose.

It seems important that both methods should work similarly and well - as previously the Jenkins task was working sub-optimally with extra rake instances and `rake test` worked correctly, while now this change has fixed the Jenkins task while breaking the user's test task.